src/main/resources/schema.sql

```diff
@@ -1,44 +1,44 @@
-DROP TABLE IF EXISTS BOOKING;
-DROP TABLE IF EXISTS USER;
-DROP TABLE IF EXISTS BOOKABLE;
-DROP TABLE IF EXISTS LOCATION;
+DROP TABLE IF EXISTS booking;
+DROP TABLE IF EXISTS user;
+DROP TABLE IF EXISTS bookable;
+DROP TABLE IF EXISTS location;
 
-CREATE TABLE USER (
-    USER_ID            VARCHAR(36) NOT NULL,
-    PRIMARY KEY        (USER_ID),
-    EXTERNAL_USER_ID   VARCHAR(128) NOT NULL,
+CREATE TABLE user (
```

Comment:
msyql & hibernate schema validation seems to be case sensitive, although nothing else seems to be...

Reply:
Weird.  I actually like lower-case table names in case of MySql case-sensitive installation (hate having to type capital letter table names in queries).